tests/unit/disco/components/TestAddon.js

```diff
@@ -458,6 +459,36 @@ describe(__filename, () => {
       const compatError = root.find(AddonCompatibilityError);
       expect(compatError.prop('reason')).toEqual(reason);
     });
+
+    it('renders an corrupt file error overlay', () => {
```

Comment:
I need help in this test case actually

Reply:
I don't think you need to add a test case for the disco pane.